### DISCUSSION TITLE
Discussions 이용하기

### DISCUSSION THREAD

**Discussion by wonderlandpark** [koreanbots/koreanbots#130] [일반] [answered]
`koreanbots/koreanbots` 레포에 깃허브 베타 기능인 Discussions가 추가되었습니다.

질문이 있으시거나, 제안사항이 있으시다면 Discussions을 이용해주세요!

**Comment by wonderlandpark** [comment, marked as answer]
버그는 [이슈](https://github.com/koreanbots/koreanbots/issues)를 이용해주세요!

**Comment by wonderlandpark** [comment]
Discussions 기능이 신기하기에 뭔가 테스트해보고 싶으신 분은 해당 토론에서 아무거나 써보셔도 됩니다!

**Reply by minibox24** [reply]
우와

> 우와

- [ ] 우와

**우와**

`우와`

1. 우
2. 와

**Reply by paring-chan** [reply]
우와

**Reply by dfdddfd** [reply]
ㄷ

**Reply by paring-chan** [reply]
ㄷㄷㄷㄷ

**Reply by samsunghappytree123** [reply]
# 와우
+ 아주 좋은 기능이군요
![image](https://koreanbots.dev/logo.png)

**Comment by ghostdeleted** [comment]
v3 개발 일정이 어떻게 되시나요

**Reply by wonderlandpark** [reply]
> v3 개발 일정이 어떻게 되시나요

off-topic

**Comment by wonderlandpark** [comment]
더 이상 베타가 아니기에 잠금처리하도록 하겠습니다
